examples/quickstart.ipynb

```diff
@@ -1,285 +1,331 @@
-      "cell_type": "markdown",
-      "metadata": {},
-      "source": [
-        "## Next Steps\n",
-        "\n",
-        "Now that you've created a custom agent, check out the\n",
-        "[tutorials](/langgraphjs/tutorials/) to learn\n",
-        "LangGraph by implementing different types of end-to-end workflows from RAG to\n",
-        "multi-agent swarms, or consult the\n",
-        "[how-to guides](/langgraphjs/how-tos/) for more\n",
-        "examples of how to implement different design patterns."
-      ]
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The current weather in San Francisco is as follows:\n",
+      "- Temperature: 82.0°F (27.8°C)\n",
+      "- Condition: Sunny\n",
+      "- Wind: 11.9 mph from the NW\n",
+      "- Humidity: 41%\n",
+      "- Pressure: 29.98 in\n",
+      "- Visibility: 9.0 miles\n",
+      "- UV Index: 6.0\n",
+      "\n",
+      "For more details, you can visit [Weather in San Francisco](https://www.weatherapi.com/).\n",
+      "The current weather in New York is as follows:\n",
+      "- Temperature: 84.0°F (28.9°C)\n",
+      "- Condition: Sunny\n",
+      "- Wind: 2.2 mph from SSE\n",
+      "- Humidity: 57%\n",
+      "- Pressure: 29.89 in\n",
+      "- Precipitation: 0.01 in\n",
+      "- Visibility: 9.0 miles\n",
+      "- UV Index: 6.0\n",
+      "\n",
+      "For more details, you can visit [Weather in New York](https://www.weatherapi.com/).\n"
+     ]
     }
-  ],
-  "metadata": {
-    "kernelspec": {
-      "display_name": "TypeScript",
-      "language": "typescript",
-      "name": "tslab"
-    },
-    "language_info": {
-      "codemirror_mode": {
-        "mode": "typescript",
-        "name": "javascript",
-        "typescript": true
-      },
-      "file_extension": ".ts",
-      "mimetype": "text/typescript",
-      "name": "typescript",
-      "version": "3.7.2"
+   ],
+   "source": [
+    "// ReActAgent.ts\n",
+    "\n",
+    "// IMPORTANT - Add your API keys here. Be careful not to publish them.\n",
+    "process.env.OPENAI_API_KEY = \"sk-...\";\n",
+    "process.env.TAVILY_API_KEY = \"tvly-...\";\n",
+    "\n",
+    "import { TavilySearchResults } from \"@langchain/community/tools/tavily_search\";\n",
+    "import { ChatOpenAI } from \"@langchain/openai\";\n",
+    "import { MemorySaver } from \"@langchain/langgraph\";\n",
+    "import { HumanMessage } from \"@langchain/core/messages\";\n",
+    "import { createReactAgent } from \"@langchain/langgraph/prebuilt\";\n",
+    "\n",
+    "// Define the tools for the agent to use\n",
+    "const agentTools = [new TavilySearchResults({ maxResults: 3 })];\n",
+    "const agentModel = new ChatOpenAI({ temperature: 0 });\n",
+    "\n",
+    "// Initialize memory to persist state between graph runs\n",
+    "const agentCheckpointer = new MemorySaver();\n",
+    "const agent = createReactAgent({\n",
+    "  llm: agentModel,\n",
+    "  tools: agentTools,\n",
+    "  checkpointSaver: agentCheckpointer,\n",
```

Comment:
Oh man we should really alias this... Want to make a PR?

Reply:
Sure! Created #455 to track it (+ the corresponding quickstart update) and assigned it to myself.